boards/nxp/imx8mm_evk/imx8mm_evk_mimx8mm6_a53_ramconsole_defconfig

```diff
@@ -0,0 +1,25 @@
+#
+# Copyright 2024 NXP
+#
+# SPDX-License-Identifier: Apache-2.0
+
+# ARM Options
+CONFIG_AARCH64_IMAGE_HEADER=y
```

Comment:
Any option here that exists in `imx8mm_evk_mimx8mm6_a53_defconfig` is not needed, applies to the other commits too

Reply:
Indeed, thanks!

Reply:
updated.